ilastik/workflows/tracking/common.py

```diff
@@ -0,0 +1,16 @@
+import sys
+
+
+class DivisionClassifierLabelNames:
```

Comment:
Maybe do something more generic?

```python
@dataclass(frozen=True)
class PluralInflector:
    zero: str
    one: str
    many: str

    def __getitem__(self, i: SupportsInt) -> str:
        i = int(i)
        if i < 0:
            raise ValueError("cannot inflect negative numbers")
        if i == 0:
            return self.zero
        return f"{i} {self.one if i == 1 else self.many}"

    __call__ = __getitem__


DIVISION_CLASSIFIER_LABEL_NAMES = PluralInflector(zero="False Detection", one="Object", many="Objects")
```

Reply:
I like the solution, but I don't really see it used anywhere else, then the explicit implementation is easier to grasp imo.

Reply:
I ended up mixing and matching a bit...